src/libecalc/presentation/yaml/mappers/fluid_mapper.py

```diff
@@ -91,54 +91,45 @@
 }
 
 
-def fluid_model_mapper(model_config: Dict, input_models: Dict[str, Any], resources: Resources):
-    fluid_model_type = model_config.get(EcalcYamlKeywords.models_type_fluid_model_type)
+def fluid_model_mapper(model_config: YamlFluidModel, input_models: Dict[str, Any], resources: Resources):
+    fluid_model_type = model_config.fluid_model_type
     mapper = _fluid_model_map.get(fluid_model_type)
     if mapper is None:
         raise ValueError(f"Fluid model type {fluid_model_type} not supported")
     return mapper(model_config=model_config)
 
 
-def _predefined_fluid_model_mapper(model_config: Dict) -> FluidModel:
-    predefined_composition_type = model_config.get(EcalcYamlKeywords.models_type_fluid_predefined_gas_type)
-    eos_model_type = model_config.get(EcalcYamlKeywords.models_type_fluid_eos_model)
+def _predefined_fluid_model_mapper(model_config: YamlPredefinedFluidModel) -> FluidModel:
+    predefined_composition_type = model_config.gas_type
+    eos_model_type = model_config.eos_model
     return FluidModel(
         eos_model=_eos_model_mapper.get(eos_model_type),
         composition=_predefined_fluid_composition_mapper.get(predefined_composition_type),
     )
 
 
 def _composition_fluid_model_mapper(
-    model_config: Dict,
+    model_config: YamlCompositionFluidModel,
 ) -> FluidModel:
-    user_defined_composition = model_config.get(EcalcYamlKeywords.composition)
+    user_defined_composition = model_config.composition
```

Comment:
raise Exception if H2O is used instead of silently ignore it? Or do we have a check for "allowed" fluids any place?

Reply:
We are validating the composition. So a generic "Key not allowed" will be given

Reply:
Are we bumping to 9 because of this?

Reply:
not at all. but just to give users a proper heads up in case they need to change their models and hopefully the "goto approach" is check migraiton guide ...